webrender/res/ps_border_corner.glsl

```diff
@@ -408,7 +396,7 @@ void main(void) {
     // Select color based on side of line. Get distance from the
     // reference line, and then apply AA along the edge.
     float ld = distance_to_line(vColorEdgeLine.xy, vColorEdgeLine.zw, local_pos);
-    float m = smoothstep(-aa_range, aa_range, ld);
+    float m = distance_aa(aa_range, -ld);
```

Comment:
shouldn't this be `1.0 - distance_aa(aa_range, ld)`?

Reply:
At a certain point my brain decided that it was equivalent, but now I am very tired and counting on the test run (and/or a good night of sleep) to help coming up with a more definitive answer to this question.

Reply:
Actually, I think you are correct here. Will see the try push and merge if it's ok.